### PR TITLE
add tests for CLI usage

### PR DESCRIPTION
**Pull request by marcospb19**

### CODE REVIEW

**Comment by figsoda** [comment]
Aren't we already testing most of them in the integration tests?

**Comment by marcospb19** [comment]
Probably, but it didn't cover #485 case, I wrote these tests while trying to solve that issue.

**Comment by figsoda** [comment]
yeah sounds good to me then

File: src/cli/args.rs [hunk 1]
```diff
@@ -97,3 +97,158 @@ pub enum Subcommand {
         tree: bool,
     },
 }
+
+#[cfg(test)]
+impl Default for CliArgs {
```
Comment (figsoda) [inline src/cli/args.rs:102]: I think we can just put this in a function inside `mod tests`

**Comment by marcospb19** [comment]
I don't plan on removing your property tests, they're useful and also provide us some fuzzing, but I might add some specific tests in the future (the objective is not to be redundant, but I might accidentally get redundant sometimes).

Talking about our proptests, can you tell why this run failed?

https://github.com/ouch-org/ouch/actions/runs/6113846588/job/16594167075

Mmmmm that's the bad thing about only having tests with a scope so broad, investigating can be very confusing.

**Comment by figsoda** [comment]
From the error message, it looks like the runner might have run out of memory possibly due to the fact that we are using cross and the combination of compression methods happened to take more memory than usual

**Comment by marcospb19** [comment]
> From the error message, it looks like the runner might have run out of memory possibly due to the fact that we are using cross and the combination of compression methods happened to take more memory than usual

That's great to hear :D , do you suggest we do something in order to reduce the chance of getting another false-positive?

**Comment by figsoda** [comment]
Maybe trying to decrease the number of extensions, or decrease the maximum size of the file

File: src/cli/args.rs (hunk 1 above)
Comment (marcospb19) [reply src/cli/args.rs]: :+1: changed and force-pushed.

**Review by figsoda — APPROVED** [review]

**Pull request merged**